draft-ietf-tls-dtls13.md

```diff
@@ -350,6 +355,9 @@ legacy_record_version
   It MUST be ignored for all purposes. See {{TLS13}}; Appendix D.1
   for the rationale for this.
 
+legacy_epoch
```

Comment:
I think you can drop ```legacy``` here. We don't use it for DTLSCiphertext

Reply:
Works for me!